src/commands/compile.ts

```diff
@@ -337,6 +337,15 @@ export async function namespaceCompile(askFlags = false): Promise<any> {
   if (!config("conn").active) {
     throw new Error(`No Active Connection`);
   }
+  const confirm = await vscode.window.showWarningMessage(
+    `Compiling all files in namespace '${api.ns}' is expensive! Are you sure you want to proceed?`,
```

Comment:
```suggestion
    `Compiling all files in namespace '${api.ns}' might be expensive. Are you sure you want to proceed?`,
```

Reply:
I like your change but I merged the PR before you made it. Can you commit it directly?